src/index.ts

```diff
@@ -24,15 +24,15 @@ const impactEngine = async () => {
   const options = parseArgs();
 
   if (options) {
-    const {inputPath, outputPath, paramPath} = options;
+    const {inputPath, outputPath, paramPath, stdout} = options;
```

Comment:
Instead of adding a fourth argument here, can you add `options` which includes `outputPath` and `stdout`? This will be more extensible if we want to add more features later.